js/zb.js

```diff
@@ -14,7 +14,14 @@ module.exports = class zb extends Exchange {
             'id': 'zb',
             'name': 'ZB',
             'countries': [ 'CN' ],
-            'rateLimit': 100,
+            // previously rateLimit = 100
+            // Trading and Margin 10 000 per minute (IP) => 10000 / 60 = 166.66666... per second => rateLimit = 1000/166.66666 = 6
```

Comment:
This line looks a bit wrong to me: per minute means divide by 60000, not 60, right ?

Reply:
Maybe that could explain the rate limiting issues you described in the initial post.

Reply:
@kroitor if the rateLimit value is the average number of milliseconds between requests, and there are 10 000 requests per minute, that makes it ~166.666... requests per second, then dividing 1000ms by this number leads to 6ms between requests

the rateLimit property is meant to be ms between requests on average for the cheapest endpoint right?

Reply:
> @kroitor if the rateLimit value is the average number of milliseconds between requests, and there are 10 000 requests per minute, that makes it ~166.666... requests per second, then dividing 1000ms by this number leads to 6ms between requests
> 
> the rateLimit property is meant to be ms between requests on average for the cheapest endpoint right?

Yep, that's right, I've misinterpreted it. No worries.